xdmod-appkernels.spec.in

```diff
@@ -51,6 +51,8 @@ rm -rf $RPM_BUILD_ROOT
 %{_libdir}/xdmod/
 %{_datadir}/xdmod/
 %{_docdir}/%{name}-%{version}__PRERELEASE__/
+%{_docdir}/%{name}-%{version}/
+%{_sysconfdir}/modules.d/appkernels.json
 
 %config(noreplace) %{_sysconfdir}/xdmod/*.d/appkernels.ini
```

Comment:
Need to not have a wildcard here as it could mark modules.d/appkernels.json as a config noreplace (which is wrong)./